db/c_test.c

```diff
@@ -1243,18 +1243,24 @@ int main(int argc, char** argv) {
     CheckValue(err, NULL, &value, size);
     value = rocksdb_writebatch_wi_get_from_batch_and_db(wbi, db, roptions,
                                                         "foo", 3, &size, &err);
+    const char* pvalue;
     CheckValue(err, "hello", &value, size);
     p = rocksdb_writebatch_wi_get_pinned_from_batch_and_db(wbi, db, roptions,
                                                            "foo", 3, &err);
-    value = rocksdb_pinnableslice_value(p, &size);
-    CheckValue(err, "hello", &value, size);
+    pvalue = rocksdb_pinnableslice_value(p, &size);
```

Comment:
perhaps?
```
const char* pvalue = rocksdb_pinnableslice_value(p, &size);
```

Reply:
I tried that but I reuse it later so it felt more appropriate to declare it separately from the first assignment. Maybe that only makes sense on my head :-)